src/main/java/org/fundacion/jala/converter/controller/VideoConverterController.java

```diff
@@ -66,10 +67,19 @@ public String uploadFile(final @RequestParam("file") MultipartFile file,
             throws IOException, InterruptedException {
         LOGGER.info("start");
         parameterOutputChecksum = ChecksumFacade.getChecksum(checksum, file);
-        String outputFilename = ConverterFacade.getVideoConverter(
-                new VideoParameter(parameterOutputChecksum.getOutputFilename(), outputFormat, resolution, thumbnail,
-                        frameRate, width, height, audio));
-        extractMetadata(metadata, outputFilename, fileStorageService);
+        String outputFilename = null;
+        try {
+            outputFilename = ConverterFacade.getVideoConverter(
+                    new VideoParameter(parameterOutputChecksum.getOutputFilename(), outputFormat, resolution, thumbnail,
+                            frameRate, width, height, audio));

```

Comment:
Lines 73 and 74 should have the same indentation.

Reply:
The editor indentates it in that way

Reply:
It is good then.